src/evict.c

```diff
@@ -242,7 +242,7 @@ void evictionPoolPopulate(int dbid, dict *sampledict, dict *keydict, struct evic
         /* Try to reuse the cached SDS string allocated in the pool entry,
          * because allocating and deallocating this object is costly
          * (according to the profiler, not my fantasy. Remember:
-         * premature optimizbla bla bla bla. */
+         * premature optimizable bla bla bla. */
```

Comment:
i think the intent was "optimize" not "optimizable"

Reply:
Thank you for your review.
Yes, `optimize` is right, fixed.
Unless the same line is changed, `cherry-pick` will not conflict, comments are rarely modified, so it can be merged with almost no impact.